src/promnesia/sources/logseq.py

```diff
@@ -0,0 +1,12 @@
+import os.path
+
+def logseq_replacer(path: str, root: str) -> str:
+    if not path.startswith("editor://") or not path.endswith('.md'):
```

Comment:
I guess for logseq also `.org` is possible? probably worth including here

Reply:
yepp